src/main/asciidoc/transactions.adoc

```diff
@@ -33,82 +41,257 @@ static class Config extends AbstractCouchbaseConfiguration {
+were meant to define a rule for all checked exceptions. With more unique
+exception names such as "BaseBusinessException" there is likely no
+need to use the fully qualified class name for the exception pattern. Furthermore,
+rollback rules defined via patterns may result in unintentional matches for
+similarly named exceptions and nested classes. This is due to the fact that a
+thrown exception is considered to be a match for a given pattern-based rollback
+rule if the name of thrown exception contains the exception pattern configured
+for the rollback rule. For example, given a rule configured to match against
+"com.example.CustomException", that rule will match against an exception
+named com.example.CustomExceptionV2 (an exception in the same package as
+CustomException but with an additional suffix) or an exception named
+com.example.CustomException$AnotherException (an exception declared as
+a nested class in CustomException).
+ 
+For specific information about the semantics of other attributes in this
+annotation, consult the org.springframework.transaction.TransactionDefinition
+and org.springframework.transaction.interceptor.TransactionAttribute javadocs.
+ 
+=== Transaction Management
+ 
+This annotation commonly works with thread-bound transactions managed by a
+org.springframework.transaction.PlatformTransactionManager, exposing a
+transaction to all data access operations within the current execution thread.
+Note: This does NOT propagate to newly started threads within the method.
+
+Couchbase does not supply a reactive transaction manager, therefore it cannot support
+@Transactional on reactive methods. Couchbase provides reactive transaction support
+via the CouchbaseTransactionalOperator. 
 
 .Transaction Access
 ====
 [source,java]
 ----
-@Autowired
-Transactions transactions;
+// this object must be provided by the Spring framework. By @Autowire or from ac.getBean().
+// If created directly with it's constructor, the method interceptor will not be called.
+@Autowired PersonService personService;
+
+    Person p = personService.savePerson(WalterWhite);
+
 
-@Autowired
-CouchbaseClientFactory couchbaseClientFactory;
+@Service
+@Component
+public class PersonService {
+
+    final CouchbaseOperations personOperations;
+
+    public PersonService(CouchbaseOperations ops) {
+        personOperations = ops;
+    }
+
+    @Transactional(transactionManager = BeanNames.COUCHBASE_TRANSACTION_MANAGER) // default, shown for clarity
+    public Person replacePerson(String id) {
+        Person p = personOperations.findById(Person.class).one(id);
+        p.setFirstname(p.getFirstname()+" Junior");
+        return personOperations.replaceById(Person.class).one(p);
+    }
 
-public void doSomething() {
- transactions.run(ctx -> {
-  ctx.insert(couchbaseClientFactory.getDefaultCollection(), "id", "content");
-  ctx.commit();
- });
 }
 ----
 ====
 
-== Object Conversions
+== Transactions with CouchbaseTransactionalOperator
```

Comment:
As above, and not to harp on it, but I think we should ditch mention of this, or at least push it to an 'for advanced users' section.  It doesn't add anything to us at all, IMO.

Reply:
Maybe it's here only because you were under the impression it was the only way to do reactive transactions?